eng/common-tests/ci.yml

```diff
@@ -8,7 +8,6 @@ trigger:
       - hotfix/*
   paths:
     include:
-      - eng/common/scripts/*
```

Comment:
Why are you removing this trigger?

Reply:
Sorry, my bad. For a moment I thought it should be removed because there should be no tests in this dir, but obviously it should be included because the test scripts should kick-off upon any change to this dir.

Reply:
@weshaggard IIUC this entire code is currently being ignored anyway due to ADO UI override. I am going to remove the override:

![image](https://user-images.githubusercontent.com/4429827/222791817-f92fa1bb-4f1a-4c04-ba92-ce8d53df17cc.png)